contracts/api3-server-v1/Api3Market.sol

```diff
@@ -121,7 +119,17 @@ contract Api3Market is HashRegistry, ExtendedSelfMulticall, IApi3Market {
     /// pointing at this contract.
     /// @param owner_ Owner address
     /// @param proxyFactory_ ProxyFactory contract address
-    constructor(address owner_, address proxyFactory_) HashRegistry(owner_) {
+    /// @param maximumSubscriptionQueueLength_ Maximum subscription queue
+    /// length
+    constructor(
+        address owner_,
+        address proxyFactory_,
+        uint256 maximumSubscriptionQueueLength_
+    ) HashRegistry(owner_) {
+        require(
```

Comment:
Maybe also enforce 10 as the maximum value?

Reply:
I don't really see the value in locking ourselves (the only ones deploying this contract) with a fixed value other than us making a mistake. I would prefer to just add more details to the `@param` comment

Reply:
I also agree with Santiago here.

Reply:
Yeah I agree, which is why I omitted it in the first place